docs/cf_remote_demo.sh

```diff
@@ -5,27 +5,21 @@ set -x
 
 SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
 
-cf-remote spawn --platform ubuntu-18-04-x64 --count 1 --name hub --role hub
+cf-remote spawn --platform ubuntu-22-04-x64 --count 1 --name hub --role hub
 cf-remote spawn --platform ubuntu-18-04-x64 --count 1 --name ubuntu18 --role client
 cf-remote spawn --platform ubuntu-16-04-x64 --count 1 --name ubuntu16 --role client
-cf-remote spawn --platform centos-6-x64 --count 1 --name centos6 --role client
 cf-remote spawn --platform centos-7-x64 --count 1 --name centos7 --role client
-cf-remote spawn --platform debian-9-x64 --count 1 --name debian9 --role client
-cf-remote spawn --platform rhel-6-x64 --count 1 --name rhel6 --role client
+cf-remote spawn --platform debian-10-x64 --count 1 --name debian10 --role client
 cf-remote spawn --platform rhel-7-x64 --count 1 --name rhel7 --role client
 
 cf-remote --version master install --demo --bootstrap hub --hub hub
 
-cf-remote deploy --hub hub "$SCRIPTPATH/../../masterfiles"
+# cf-remote deploy --hub hub "$SCRIPTPATH/../../masterfiles"
```

Comment:
why comment this out? why not just remove it?

Reply:
The commit message explains why it's being commented out.

I am not removing it completely because it's useful for me to sometimes deploy the masterfiles i have in my masterfiles folder. For example if I am testing a new masterfiles policy change. So then I remove those 2 characters. I quite often make small edits to this script before running it (for example changing a count, a platform, or the `--version` argument). All of these things could be put in command line arguments, of course, but I don't think it's really necessary / advantageous to do so. Now it's mostly just a list of copy-pastable shell commands, nothing too complicated.